extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -92,6 +92,17 @@ const mode = {
   GEO_OVERRIDE: 2, //  We've been overriden in test by #amp-geo=xx
 };
 
+/**
+ * @typedef {{
+ *   ISOCountry: string,
+ *   matchedISOCountryGroups: !Array<string>,
+ *   allISOCountryGroups: !Array<string>,
+ *   isInCountryGroup: GEO_IN_GROUP,
+ *   ISOCountryGroups: !Array<string>
+ * }}
+ */
+export const GeoDef = {};
```

Comment:
Who is importing the `GeoDef` here?

Reply:
I'd like to understand that too.

Reply:
sorry bout that, I've fixed the references now. GeoDef is the return of `addToBody_` and it is the type you get from the `geoForDocOrNull` service locator